infra/init.sh

```diff
@@ -8,6 +8,7 @@ helm repo add bitnami-labs https://bitnami-labs.github.io/sealed-secrets/
 helm repo add cert-manager https://charts.jetstack.io
 helm repo add ingress-nginx https://kubernetes.github.io/ingress-nginx
 helm repo add metallb https://metallb.github.io/metallb
+helm repo add oauth2-proxy https://oauth2-proxy.github.io/manifests
```

Comment:
we should add this as a dependency to `base` instead of its own installation using `--values`, similar to how mongodb is a dependency to the `mongo` chart